src/app/package-json.js

```diff
@@ -107,7 +108,10 @@ const packageJSON = (current, context) => {
       ]),
 
       'build:js:browserify': scriptify([
-        'browserify . -s %s -o dist/%s.js'
+        'browserify .',
+        '-t babelify',
+        '-s ' + context.functionName,
```

Comment:
Is `context.pluginName` not what we want the standalong name to be? How does `context.functionName` compare?